### PR TITLE
Pia 1937 add capture sdk

### PR DESCRIPTION
**Pull request by a-szotyori**
Added the Capture SDK along with the default network implementation and the example apps.

### CODE REVIEW

File: bank-api-library/LICENSE.md [hunk 1]
```diff
@@ -1 +1 @@
-Moved to http://developer.gini.net/gini-mobile-android/bank-api-library/html/license.html
\ No newline at end of file
+Moved to http://developer.gini.net/gini-mobile-android/bank-api-library/library/html/license.html
```
Comment (zladzeyka) [inline bank-api-library/LICENSE.md:1]: I'm trying to use " https" in all links where i can notice it ( warning not secure looks a bit disturbing)😊

Comment (a-szotyori) [reply bank-api-library/LICENSE.md:1]: That's a great point! I will replace all links to use https.

**Review by zladzeyka — APPROVED** [review]
🎉👍🏻 Great job!

File: bank-api-library/LICENSE.md (hunk 1 above)
Comment (a-szotyori) [reply bank-api-library/LICENSE.md:1]: I updated it in PR #11.

**Pull request merged**